Main.py

```diff
@@ -47,8 +47,10 @@ async def on_command_error(self, context, exception):
         logger.error("Encountered traceback:\n" + tb)
 
 
-bot = MiniCrosswordBot(command_prefix=CMD_PREFIX)
-
+intents = discord.Intents.default()
+intents.members = True
+intents.presences = True
```

Comment:
Could you elaborate on why we need these new intents? I can't run the bot with the exact same Discord Developer Portal settings anymore. 

https://discordpy.readthedocs.io/en/latest/intents.html#privileged-intents